peer.go

```diff
@@ -224,6 +268,14 @@ func (p *Peer) GetConnection(ctx context.Context) (*Connection, error) {
 	return c, nil
 }
 
+// NumInbound returns the number of inbound connections to this node.
```

Comment:
this change is still there even though there's no actual change, all that's happened is that it's moved. please revert